src/main/ipv-stub/ipv-authorize.ts

```diff
@@ -63,18 +64,134 @@ async function get(
     return errorResponse(500, "Decoding failed: " + message);
   }
 
+  const jsonBody = JSON.parse(decodedPayload);
+
   return {
     statusCode: 200,
     headers: { "Content-Type": "text/html" },
     body: renderPage(
-      `<h1 class="govuk-heading-l">Hello World from the IPV authorize lambda</h1>
+      `<h1 class="govuk-heading-l">IPV authorize lambda stub</h1>
   <h3 class="govuk-heading-s">Decrypted JAR header:</h3>
-  <p class="govuk-body">${JSON.parse(decodedHeader)}</p>
+  <dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
```

Comment:
Maybe this form should live in a separate file as it bloats the lambda quite a lot